docs/agent-api.asciidoc

```diff
@@ -68,6 +68,21 @@ Use `isStarted()` to check if the agent has already started.
 Returns `true` if the agent has started, 
 otherwise returns `false`.
 
+[[apm-set-framework]]
+==== `apm.setFramework(options)`
+
+[small]#Added in: v2.8.0#
+
+* `options` {uri-object}[`<Object>`]
+** `name` {uri-string}[`<string>`] Framework name.
```

Comment:
See comment below for an explanation. If you go with that recommendation, this line becomes:

```suggestion
** `name` +{uri-string}+ Framework name.
```